### PR TITLE
Sample app added

### PR DESCRIPTION
**Pull request by bmourat**

### CODE REVIEW

File: sampleapp-tvos-swift.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				MTL_ENABLE_DEBUG_INFO = YES;
+				ONLY_ACTIVE_ARCH = YES;
+				SDKROOT = appletvos;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				TARGETED_DEVICE_FAMILY = 3;
+				TVOS_DEPLOYMENT_TARGET = 10.2;
```
Comment (jaeklim) [inline sampleapp-tvos-swift.xcodeproj/project.pbxproj]: Minimum target is 10.0

File: sampleapp-tvos-swift.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				MTL_ENABLE_DEBUG_INFO = NO;
+				SDKROOT = appletvos;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				TARGETED_DEVICE_FAMILY = 3;
+				TVOS_DEPLOYMENT_TARGET = 10.2;
```
Comment (jaeklim) [inline sampleapp-tvos-swift.xcodeproj/project.pbxproj]: Minimum target is 10.0

**Review by jaeklim — CHANGES_REQUESTED** [review]
Cannot build the app.
```ld: framework not found Pods_sampleapp_tvos_swift
clang: error: linker command failed with exit code 1 (use -v to see invocation)```

**Comment by jaeklim** [comment]
Looks the error that I mentioned above is caused by wrong cocoapods configuration. Will keep this PR as is until we release tvOS SDK.

File: Podfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+target 'sampleapp-tvos-swift' do
```
Comment (iamclement) [inline Podfile:3]: We should specify the min TV OS SDK version in the Podfile too, I think it's 10.0

**Review by iamclement — CHANGES_REQUESTED** [review]
Just one minor thing

**Review by iamclement — APPROVED** [review]

**Review by jaeklim — APPROVED** [review]

**Pull request merged**